### PR TITLE
#78 - Make the input only for numbers in student id field

### PR DESCRIPTION
**Pull request by ImmortalIdiot**
The user can input only digits in student id input fields. U can see this feature on preview below
image
image

### CODE REVIEW

File: app/src/main/res/layout/fragment_registration.xml [hunk 1]
```diff
@@ -163,13 +163,13 @@
                     android:layout_width="match_parent"
                     android:layout_height="match_parent"
                     android:digits="0123456789"
+                    android:inputType="numberPassword"
                     android:background="@drawable/white_rounded_background"
                     android:hint="@string/student_id"
-                    android:imeOptions="actionDone"
+                    android:imeOptions="flagNavigateNext"
```
Comment (Xandr070) [inline app/src/main/res/layout/fragment_registration.xml:169]: The next button cannot be used to go further. This works correctly in the first image, but not in the second
photo_2023-12-20_16-54-02

photo_2023-12-20_16-54-07

Comment (Xandr070) [inline app/src/main/res/layout/fragment_registration.xml:169]: This parameter is ignored on my device, but it works correctly in the authorization fragment. The first image shows an option that, when you click on the button, moves to the next field, but not in the second one. Please check it again on your device and give feedback
photo_2023-12-20_16-54-02
photo_2023-12-20_16-54-07

Comment (ImmortalIdiot) [reply app/src/main/res/layout/fragment_registration.xml:169]: Yes, in my previous comment, the buttons are also different

Comment (ImmortalIdiot) [reply app/src/main/res/layout/fragment_registration.xml:169]: I fixed it, check please

**Review by Xandr070 — APPROVED** [review]
Everything is working correctly now

**Review by NN171 — APPROVED** [review]

**Pull request merged**